### PR TITLE
0.3.2

### PR DESCRIPTION
**Pull request by jnblanchard**
Confirmation flow for iOS native pay. 

Doc changes, and testable keys

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: stripe_payment
 description: A Flutter plugin to integrate the stripe plugin for iOS and Android. Currently only adding a credit card as payment method is implemented.
 version: 0.3.1
```
Comment (jonasbark) [inline pubspec.yaml]: can you adjust the version number?

File: README.md [hunk 2]
```diff
@@ -1,62 +1,81 @@
...
+
+## Setup
+
+Using **publishable key** and **merchant identifier**, ready the app for collecting user payment information. This needs to be done prior to opening card input or a native payment sheet.
 
-To set your publishable key set:
 ```dart
-import 'package:stripe_payment/stripe_payment.dart';
-StripePayment.setSettings(StripeSettings(publishableKey: "pk_test_", merchantIdentifier: "Test", androidProductionEnvironment: false));
+var settings = StripeSettings(publishableKey:"pk_test_yZuUz6SqmH4lA7SrlAvYCh003MvJiJlR", merchantIdentifier: "merchant.stripe-example", androidProductionEnvironment: false);
```
Comment (jonasbark) [inline README.md]: you wouldn't want to have that key public, right? :D

**Comment by ayushin** [comment]
```E/flutter: [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method completeNativePay on channel stripe_payment)
```

It would be good to have a stub for this in Android to avoid this.

File: pubspec.yaml (hunk 1 above)
Comment (jnblanchard) [reply pubspec.yaml]: You got it, changed in 'bump yaml version'

File: README.md (hunk 2 above)
Comment (jnblanchard) [reply README.md]: I don't really mind as long as they cant make it to my secret key and start making charges.

I changed the ID, and key ever so slightly to not be the ones I use in the example too.

**Pull request merged**